### PR TITLE
docs: add the license of logo

### PR DESCRIPTION
**Pull request by suzuki-shunsuke**
Follow up #403 #405

I have generated the code with [Choose a license](https://creativecommons.org/choose/?lang=en).

http://creativecommons.org/licenses/by-nc-nd/4.0/

### CODE REVIEW

**Comment by suzuki-shunsuke** [comment]
@vadasambar Could you check if the license is appropriate?

**Comment by suzuki-shunsuke** [comment]
I think you can distinguish the license of code and logo.

**Comment by vadasambar** [comment]
Sorry for the late reply. I will take a look at this today. :pray:

File: logo/README.md [hunk 1]
```diff
@@ -18,3 +18,7 @@ It has been built with open source:
 * Something that incorporates the image of water (aqua) and package
 * Minimalistic: should express the idea with less design components
 * Scalable: should look good irrespective of the scale (favicon, t-shirts, print-outs and everything else)
+
+## License of Logo
+
+<a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License</a>.
```
Comment (vadasambar) [inline logo/README.md:24]: This is pretty close to [the official Kubernetes logo license](https://github.com/kubernetes/kubernetes/blob/master/logo/LICENSE).
I think in the future, we might want to remove the `NonCommercial` (if a company wants to use Aqua) and `NoDerivatives` (if someone wants to use modified Aqua logo for their registry) part but it should be okay to keep it for now.

Comment (vadasambar) [reply logo/README.md:24]: Otherwise, LGTM. Thanks for researching about license!

**Review by vadasambar — APPROVED** [review]

File: logo/README.md (hunk 1 above)
Comment (suzuki-shunsuke) [reply logo/README.md:24]: Thank you for your review!

**Pull request merged**